fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -64482,6 +64572,678 @@ is_a: FBbt:00057012 ! female germline cell
 relationship: develops_from FBbt:00004877 ! cystocyte
 relationship: part_of FBbt:00004866 ! germarium
 
+[Term]
```

Comment:
Change to T1/2/3

Reply:
Regarding thoracic segments: I have never heard anyone in the larval community actually refer to the meta- meso- etc. thoracic segments of the VNC.... so I really think that should be kept as T1-3.

Reply:
OK so 

centrolateral neuropil of larval mesothoracic segment
>
name: centrolateral neuropil of larval T1 segment
synonym: centrolateral neuropil of larval mesothoracic segment
synonym: centrolateral neuropil of larval thoracic segment 1

 These terms are autogenerated using segment names in the ontology so will need to futz with pattern spec.   To be consistent,  should I make the abdominal ones A1-8 ?

Reply:
sounds great, yes thanks